package/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM httpd:2.4.57-alpine
+FROM httpd:alpine3.20
```

Comment:
@deepthi-mantena alpine3.20 seems like a generic tag which does not show the httpd version. I would recommend using [2.4.59-alpine3.20](https://hub.docker.com/layers/library/httpd/2.4.59-alpine3.20/images/sha256-2cf7f5391c32edf94f825e06207c82ab402e17b455e5e61075a149a238986a99?context=explore) which has a fixed version of httpd with 2.4.59